movies-manager/src/containers/main/main.tsx

```diff
@@ -49,6 +51,7 @@ function Main({
     setMovieDetails,
     ...props
```

Comment:
To complete `When new user enters the site using direct link with search parameters – show search results` you have to read data from `window.location`